src/sas/sasgui/perspectives/fitting/models.py

```diff
@@ -259,7 +259,9 @@ def findModels(self):
         """
         temp = {}
         if self.is_changed():
-            return  _findModels(dir)
+            temp =  _findModels(dir)
+            self.last_time_dir_modified = time.time()
```

Comment:
Unfortunately, this change is critically necessary.  

When _findModels reads a ".py" file, it compiles it into a ".pyc" file.  This changes the last last modified time for the directory.  The actual variable, however, is not updated, so SasView was assuming that the directory had changed since its last read and was reloading again.